lib/holidays.rb

```diff
@@ -9,44 +9,6 @@
 require 'holidays/errors'
 require 'holidays/load_all_definitions'
 
-# == Region options
```

Comment:
Yep. They were out of date in some cases and honestly I'm not a huge fan of in-code comments like this. I believe that the code should be self-documenting and, if additional doc is needed, the README should fill in the gaps.

Thoughts? Was this kind of documentation useful to you? I'm open to discussing it, although I would want to re-add it in another PR/issue so that they are up to date if we do go that route.